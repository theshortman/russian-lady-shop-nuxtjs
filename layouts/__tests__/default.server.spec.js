/**
 * @jest-environment node
 */

import { renderToString } from '@vue/server-test-utils'
import DefaultLayout from '../default.vue'


test('renders correctly default layout', async () => {
  const str = await renderToString(DefaultLayout)
  
  expect(str).toMatchSnapshot()
})