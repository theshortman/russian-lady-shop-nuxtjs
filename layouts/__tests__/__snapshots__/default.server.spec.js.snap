// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly default layout 1`] = `
"<div data-server-rendered=\\"true\\"><div class=\\"border-bottom text-danger text-center px-2\\">
    Магазин закрыт. Спасибо всем, кто у нас покупал.
  </div> <TheNavbar></TheNavbar> <div class=\\"container-lg mb-5\\"><div class=\\"row\\"><div class=\\"col-md-4 col-lg-3\\"><TheCart></TheCart> <div><CategoryList categories=\\"\\"></CategoryList></div></div> <div class=\\"col-md-8 col-lg-9\\"><Nuxt></Nuxt></div></div></div></div>"
`;
